SeleniumPracticalExercise/PageObjects/EmployeeInformationPage.cs

```diff
@@ -6,19 +6,20 @@ namespace SeleniumPracticalExercise.PageObjects
 {
     public class EmployeeInformationPage : BasePageLocal
     {
-        
+        private readonly By _actualFirstNameLocator = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");
```

Comment:
Remember, the "actual"/"expected" prefix for variables shouldn't be used for locators. It was intended only to be used around Asserts.

Reply:
Renamed them to have ResultsLocator